pom.xml

```diff
@@ -61,11 +61,25 @@
       <version>${iceberg.version}</version>
       <classifier>all</classifier>
     </dependency>
+    <!-- https://mvnrepository.com/artifact/com.esotericsoftware/kryo -->
```

Comment:
You don't need to put these links in, should be self-evident from the Maven co-ordinates.